From ef1578c9fef421a22c030d728fb5ef77bb7b86f6 Mon Sep 17 00:00:00 2001
From: Dominique Leuenberger <dimstar@opensuse.org>
Date: Wed, 8 Aug 2012 15:11:24 +0200
Subject: [PATCH] gphoto: support 2.5, but do not require it.

Using GPHOTO2 for a 2nd PKG_CHECK_MODULE does not give the expected
result, as the flag 'FOUND' is already set and the higher version
requirement will not be honored.

https://bugzilla.gnome.org/show_bug.cgi?id=681448
---
 configure.ac | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index c32d425..6ffe06d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -436,8 +436,9 @@ if test "x$enable_gphoto2" != "xno" -a \( "x$msg_hal" = "xyes" -o "x$msg_gudev"
   if test "x$msg_gphoto2" = "xyes"; then
     if test "x$use_gphoto2" = "xyes"; then
       AC_DEFINE(HAVE_GPHOTO2, 1, [Define to 1 if gphoto2 is available])
-      PKG_CHECK_MODULES(GPHOTO2, libgphoto2 >= 2.5.0,
-         AC_DEFINE(HAVE_GPHOTO25, 1, [Define to 1 if libgphoto2 2.5 is available])
+      PKG_CHECK_MODULES(GPHOTO25, libgphoto2 >= 2.5.0,
+         AC_DEFINE(HAVE_GPHOTO25, 1, [Define to 1 if libgphoto2 2.5 is available]),
+	 []
       )
     else
       if test "x$enable_gphoto2" = "xyes"; then
-- 
1.7.12

